packages/runner/src/suite.ts

```diff
@@ -215,9 +217,10 @@ function createDefaultSuite(runner: VitestRunner) {
 export function clearCollectorContext(
   filepath: string,
   currentRunner: VitestRunner,
+  file: File,
 ): void {
   if (!defaultSuite) {
-    defaultSuite = createDefaultSuite(currentRunner)
+    defaultSuite = createDefaultSuite(currentRunner, file)
```

Comment:
Maybe it's better to assign it after it's created/read here? (on line 225) If `defaultSuite` already exists in `isolate: false`, I think it will have incorrect `file` at one point